enterprise/authentication-email-password-setup.md

```diff
@@ -0,0 +1,10 @@
+# Authentication with Email Addresses and Passwords
```

Comment:
I did ask you to please also correct the filename again when i asked to update the headline:

enterprise/authentication-email-addresses-passwords-setup.md

Reply:
Missed this. Updated now.